src/clr-addons/datagrid/enum-filter/enum-filter.component.ts

```diff
@@ -51,19 +57,21 @@ export class ClrEnumFilterComponent<T extends { [key: string]: any }>
     });
   }
 
-  setPossibleValues(values: string[]) {
-    this.possibleValues = values;
+  setPossibleValues(values: (string | FilterValue)[]) {
+    this.possibleValues = values.map(v => (v instanceof Object ? v : { value: v, displayValue: v }));
     this.possibleValues.sort();
```

Comment:
this should sort the values by displayValue

Reply:
adapted sorting